### DISCUSSION TITLE
Participate in the MSIX Developer Survey '24!

### DISCUSSION THREAD

**Discussion by fiza-azmi** [Microsoft/msix-packaging#630] [General]
Hello MSIX community,

We’re inviting **developers** to contribute to the enhancement of MSIX! 

We’ve initiated a [survey](https://forms.office.com/r/pvjrGj3tGx) to gauge your experience with MSIX and discover areas for improvement. By sparing a few minutes to share your thoughts, you can play a crucial role in refining MSIX for the developer community. 

##
 **Get Started:**
 Take the survey here: [**Survey Link**](https://forms.office.com/r/pvjrGj3tGx)
## 
Thank you for being a part of the community. Your insights will help us make MSIX better for all developers! 
Want to learn more about MSIX Packaging? [Explore here](https://learn.microsoft.com/en-us/windows/msix/overview)

_The MSIX Platform Team_

**Comment by beedell-roke** [comment]
> https://github.com/microsoft/msix-packaging/discussions/630#discussion-6674412

@fiza-azmi, done. [^1] Thanks. I hope you don't mind me having accidentally filled https://github.com/microsoft/msix-packaging/discussions/594#discussioncomment-11954381 beforehand.

[^1]: [`forms.office.com/Pages/ResponseDetailPage.aspx?id=v4j5cvGGr0GRqy180BHbR53RyrInQKlHvBfVixwW0XhUMjVWUklNU1BHWDJGN1FYOVlJVkJJMDgwTC4u&rid=16&GetResponseToken=98It2AFy2xBGB-ILSE2UyjskM3NHN8e2sSRLdB85HMc`](https://forms.office.com/Pages/ResponseDetailPage.aspx?id=v4j5cvGGr0GRqy180BHbR53RyrInQKlHvBfVixwW0XhUMjVWUklNU1BHWDJGN1FYOVlJVkJJMDgwTC4u&rid=16&GetResponseToken=98It2AFy2xBGB-ILSE2UyjskM3NHN8e2sSRLdB85HMc)
